webextensions/api/tabs.json

```
@@ -607,6 +607,25 @@
               }
             }
           },
+          "attention": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": true
```

```suggestion
                  "version_added": "63"
```
version 63, source: https://bugzilla.mozilla.org/show_bug.cgi?id=1396684

To find this, I repeatedly used the "annotated diff" feature in the sidebar while searching for "attention" . I started at https://searchfox.org/mozilla-central/rev/d572999cdc8f41591ce2f4d0b13486cc9aad2123/browser/components/extensions/schemas/tabs.json#220 and ended at https://searchfox.org/mozilla-central/diff/180811af326c308b68f52ef45146644b02ffe378/browser/components/extensions/schemas/tabs.json#105